compose/config/config.py

```diff
@@ -101,6 +101,7 @@
 
 ALLOWED_KEYS = DOCKER_CONFIG_KEYS + [
     'build',
+    'cache_from',
```

Comment:
`cache_from` is under `build`, so it shouldn't be in this list. This list is only for the top level service keys